docs/private-networks/how-to/monitor/chainlens.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Use Chainlens Explorer
+sidebar_position: 7
+description: Use Chainlens Explorer on a privacy-enabled Besu network
+tags:
+  - private networks
+---
+
+# Use Chainlens Blockchain Explorer
+
+[Chainlens Blockchain Explorer](https://www.web3labs.com/chainlens) supports public and private EVM networks. This page describes how to use the free version of Chainlens with its built-in support for [privacy-enabled](../../concepts/privacy/index.md) Besu networks created using the [Developer Quickstart](../../tutorials/quickstart.md).
```

Comment:
We should stick to https://chainlens.com/ anywhere we use https://www.web3labs.com/chainlens